### ISSUE TITLE
Ccw product splash

### ISSUE THREAD

**Issue by arichiardi** [laurentpetit/ccw#758]
This absolutely NOT silly issue is going to be opened in order to collect some ideas on Counterclockwise standalone splash logo.

I had a brainstorming session with a friend of mine on how to improve it (of course improve a piece of art is always a matter of opinions) and this is what we came up with.

![_20150502_111829](https://cloud.githubusercontent.com/assets/1724788/7440550/fe9cb96c-f0bc-11e4-9b3b-2441d67c4bb8.JPG)

Of course I expect some critics, which are always how you make something better! If it is something that is appreciated and liked...I will start working on the digital version ;)

**Comment by laurentpetit** [comment]
I really like the idea !
I think that it would be interesting to add a full-text description of what CCW is and stands for.
Not as big as the logo, of course, maybe somewhere below the logo, right-indented with it, with a font and font-size and font-weight TBD, saying something short like "CounterClockWise, the Clojure IDE based on Eclipse" ?

**Comment by arichiardi** [comment]
Yes I was thinking about that and I like it. Now I just need to work with some designing tool :)))
